src/OpenTK.Mathematics/Data/Quaternion.cs

```diff
@@ -191,7 +191,7 @@ public Vector4 ToAxisAngle()
         /// <summary>
         /// Returns a copy of the Quaternion scaled to unit length.
         /// </summary>
-        /// <returns>The normalized copy.</returns>
```

Comment:
Possible loss of information. Should probably keep as "copy" instead of "Quaternion". We know it's a quaternion.

Reply:
well I'm not too sure on that I mean it's a struct, of course it's a copy

Reply:
It's more informative than Quaternion; copy semantics are an important facet of the mathematics subsystem.